internal/service/ipam/model_ipamsvc_dhcp_config.go

```diff
@@ -47,13 +46,11 @@ var IpamsvcDHCPConfigResourceSchemaAttributes = map[string]schema.Attribute{
 	"abandoned_reclaim_time": schema.Int64Attribute{
 		Optional:            true,
 		Computed:            true,
-		Default:             int64default.StaticInt64(3600),
```

Comment:
Any reason why this had to be removed?

Reply:
The `abandoned_reclaim_time` and `abandoned_reclaim_time_v6` are not supported for subnet. Having the default there causes an error in API, as the API doesn't expect the value in the field.